cypress/fixtures/singleBook.json

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
Nice add to make it an array of object to match what is returned on the GET request